release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1727,10 +1732,55 @@ If these values are not defined in a failure domain configuration, for instance
+
+* Previously, mirroring a release that used a digest instead of a tag did not work as expected. This issue has been fixed, ensuring successful mirroring for releases that use digests. (link:https://issues.redhat.com/browse/OCPBUGS-45249[*OCPBUGS-45249*])
+
+* Previously, `oci` catalogs on disk were not mirrored correctly in oc-mirror plugin v2. With this update, `oci` catalogs are now successfully mirrored. (link:https://issues.redhat.com/browse/OCPBUGS-44225[*OCPBUGS-44225*])
+
+* Previously, if you reran the `oc-mirror` command,  the rebuild of the `oci` catalog failed and an error was outputted. With this release, if you rerun the command, the wroksapce file is deleted so that failed catalog issue does not happen. (link:https://issues.redhat.com/browse/OCPBUGS-45171[*OCPBUGS-45171*])
+
+* Previously, if you ran the `oc adm node-image create` command on the first attempt, sometimes an `image can't be pulled` error message was outputted on the terminal. With this release, a patch is applied to the command where the patch adds a retry mechanism to address temporary failures when pulling the image from the release payload. (link:https://issues.redhat.com/browse/OCPBUGS-44388[*OCPBUGS-44388*])
+
+* Previously, `oci` catalogs on disk were not mirrored correctly in oc-mirror plugin v2. With this update, `oci` catalogs are now successfully mirrored. (link:https://issues.redhat.com/browse/OCPBUGS-44225[*OCPBUGS-44225*])
+
+* Previously, duplicate entries could appear in the signature `ConfigMap YAML` and `JSON` files created in `clusterresource`, leading to issues when applying them to the cluster. This update ensures that the generated files do not contain duplicates. (link:https://issues.redhat.com/browse/OCPBUGS-42428[*OCPBUGS-42428*])
+
+* Previously, API calls to Cincinnati that encountered errors did not fail immediately, potentially leading to inconsistencies. With this update, errors related to Cincinnati are now considered severe and cause an immediate failure. (link:https://issues.redhat.com/browse/OCPBUGS-38461[*OCPBUGS-38461*])
+
+* Previously, the release signature `ConfigMap` for oc-mirror plugin v2 was incorrectly stored in an archived TAR file instead of in the `cluster-resources` folder. This caused `mirror2disk` to fail. With this release. the release signature `ConfigMap` for oc-mirror plugin v2 that is in JSON format or YAML format, compatible with  oc-mirror plugin v1, now get stored in the `cluster-resources` folder. (link:https://issues.redhat.com/browse/OCPBUGS-38343[*OCPBUGS-38343*]) and (link:https://issues.redhat.com/browse/OCPBUGS-38233[*OCPBUGS-38233*])
```

Comment:
```suggestion
* Previously, the release signature `ConfigMap` for oc-mirror plugin v2 was incorrectly stored in an archived TAR file instead of in the `cluster-resources` folder. This caused `mirror2disk` to fail. With this release. the release signature `ConfigMap` for oc-mirror plugin v2 that is in JSON format or YAML format, compatible with oc-mirror plugin v1, now get stored in the `cluster-resources` folder. (link:https://issues.redhat.com/browse/OCPBUGS-38343[*OCPBUGS-38343*]) and (link:https://issues.redhat.com/browse/OCPBUGS-38233[*OCPBUGS-38233*])
```